main.tex

```diff
@@ -19,5 +19,18 @@
 
 \maketitle
 
+% 1.2.3 in Gliederung
+\section{Netzneutralität}
+
+Netzneutralität, oder `Net Neutrality' im Englischen, beschreibt ein Prinzip, nach welchem ein Internetanbieter allen Datenverkehr gleich behandelt, dass heißt ohne bestimmte Internetangebote langsamer oder teurer zu machen als andere \autocite{netzneutralität}.
+
+\medskip
+
+Indiens Vergangenheit, im Bezug auf Netzneutralität ist, anders als die der USA, noch sehr jung. So gab es vor Internet.org weder bestehende Verordnungen und auch nur einige Vorfälle welche eine Diskussion des Themas veranlassten.
```

Comment:
Wenn nicht noch auf die Netzneutralität in der USA eingegangen wird, würde ich `anders als die der USA` weglassen.